tests/scripts/mbed-build.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+# mbed-build.sh
+#
+# This file is part of Mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2019, Arm Limited, All Rights Reserved
+#
+# Purpose
+#
+# To run test builds of the mbed OS module for all specified targets.
+# Usage: mbed-build <folder to import to> <optinally string of space seperated targets>
```

Comment:
I think that this usage documentation is very incomplete. It looks like the script can also handle a few parameters from the environment variables. I think these should be documented here.